managers/tests/test_unit_model.py

```diff
@@ -1,48 +1,120 @@
+import datetime
```

Comment:
Please change it into `from datetime import datetime`

Reply:
Done